modules/mvt/src/lib/binary-vector-tile/features-to-binary.js

```diff
@@ -93,6 +94,7 @@ function fillArrays(features, firstPassData = {}, options = {}) {
         ? new Uint32Array(polygonRingsCount + 1)
         : new Uint16Array(polygonRingsCount + 1),
     positions: new PositionDataType(polygonPositionsCount * coordLength),
+    triangles: [],
```

Comment:
Would be great to update the doc of binary format.
 https://github.com/visgl/loaders.gl/blob/master/modules/gis/docs/api-reference/geojson-to-binary.md

Reply:
I've added a note to the doc